integrations/sonarqube/.port/resources/port-app-config.yaml

```diff
@@ -36,6 +36,40 @@ resources:
             coverage: .__measures[]? | select(.metric == "coverage") | .value
             mainBranch: .__branch.name
             tags: .tags
+  - kind: ga_projects
+    selector:
+      query: 'true'
+      metrics:
+        - code_smells
+        - coverage
+        - bugs
+        - vulnerabilities
+        - duplicated_files
+        - security_hotspots
+        - new_violations
+        - new_coverage
+        - new_duplicated_lines_density
```

Comment:
this is good. to avoid having 2 mappings, let's delete the project kind mapping so that for fresh installations, they will use the ga_projects